datasets/huishoudelijkafval/cluster/v2.0.0.json

```diff
@@ -75,7 +84,7 @@
         "type": "string",
         "relation": "gebieden:buurten",
         "uri": "https://www.amsterdam.nl/stelselpedia/gebieden-index/catalogus/buurt",
-        "description": "Unie­ke iden­ti­fi­ca­tie van het ob­ject"
+        "description": "Unie\u00adke iden\u00adti\u00adfi\u00adca\u00adtie van het ob\u00adject"
```

Comment:
Looks fishy: hier zitten allemaal no-break spaces in de woorden.

Misschien toch beter met die Unicode escaping.

Reply:
Scherp! Dat zijn soft-hyphens (woordafbrekingen) die je niet ziet bij het kopieren van teksten.